pom.xml

```diff
@@ -70,6 +70,12 @@
             <version>2.0.16</version>
         </dependency>
 
+        <dependency>
+            <groupId>no.nav.security</groupId>
+            <artifactId>oidc-spring-support</artifactId>
+            <version>0.2.18</version>
+        </dependency>
+
```

Comment:
Hvorfor er denne tatt inn ifm. cache?

Reply:
For test. Glemte å legge inn scope

Reply:
Ser ut som du bruker Nimbus, som er en transitiv avhengighet av oidc-spring-support. Vurder om den kan tas inn direkte.